.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U00002728Enhancement"
+labels: "Type: Enhancement"
```

Comment:
Nit: "Enhancement" is a bit broad, especially if you have "Maintenance" as a separate category.

Maybe "Feature Request" and "DevX Request"? Or consolidate them into a single template?